### PR TITLE
Fix todo issue in googlecloudmonitoringreceiver

### PR DESCRIPTION
**Pull request by abhishek-at-cloudwerx**
#### Description: 
Fix todo comments issue in googlecloudmonitoringreceiver receiver component

### CODE REVIEW

File: receiver/googlecloudmonitoringreceiver/receiver.go [hunk 1]
```diff
@@ -314,8 +314,6 @@ func (mr *monitoringReceiver) convertGCPTimeSeriesToMetrics(metrics pmetric.Metr
 		mr.metricsBuilder.ConvertSumToMetrics(timeSeries, m)
 	case metric.MetricDescriptor_DELTA:
 		mr.metricsBuilder.ConvertDeltaToMetrics(timeSeries, m)
-	// TODO: Add support for HISTOGRAM
```
Comment (dashpole) [inline receiver/googlecloudmonitoringreceiver/receiver.go:317]: I'm not sure why we are removing these.  We should keep them around until it is implemented.

**Review by atoulme — CHANGES_REQUESTED** [review]
agree, this TODO needs to be implemented.

File: receiver/googlecloudmonitoringreceiver/receiver.go (hunk 1 above)
Comment (abhishek-at-cloudwerx) [reply receiver/googlecloudmonitoringreceiver/receiver.go:317]: Yes, that's correct. 
I'll close the PR for now as it's important to implement this in the future.

**Comment by abhishek-at-cloudwerx** [comment]
Support for histograms will be needed in the future, so the TODO will remain as is.

**Comment by abhishek-at-cloudwerx** [comment]
Support for histograms will be needed in the future, so the TODO will remain as is.

**Pull request closed without merging**